lib/pbench/server/database/models/auth_tokens.py

```diff
@@ -5,13 +5,16 @@
 
 from pbench.server.database.database import Database
 
+# Convenient shortened name
+dbs = Database.db_session
```

Comment:
Disclaimers:
- I'm not necessarily advocating a change at this point
- I'm not really comfortable with justifying accepting changes that we "don't want" based on the assertion that we "promise" to correct it in the future.  (It would be better to defer the proposed change to the future point, rather than to try to make it twice with the risk that the second one might never arrive.)

Those said, as a general rule, the length and descriptiveness of a name should be directly proportional to the size of its scope.  Thus, for a block or a small function, `dbs` would be a perfectly acceptable name; however, as a variable with a module-level scope (or wider, thanks to Python's `import` mechanism), `dbs` would be unacceptably cryptic.